src/main/java/com/emc/pravega/perf/PravegaReaderWorker.java

```diff
@@ -30,60 +28,44 @@
 import io.pravega.client.stream.EventRead;
 import io.pravega.client.stream.ReinitializationRequiredException;
 
-public class PravegaReaderWorker implements Callable<Void> {
+public class PravegaReaderWorker extends ReaderWorker  {
     public final static AtomicInteger eventCount = new AtomicInteger(0);
```

Comment:
I don't think this should be public or static.

Reply:
thanks.. I have made this one as private now